airrohr-firmware/airrohr-firmware.ino

```diff
@@ -527,6 +527,15 @@ float value_SPS30_N2 = 0.0;
 float value_SPS30_N3 = 0.0;
 float value_SPS30_N4 = 0.0;
 float value_SPS30_TS = 0.0;
+
+float last_value_BMP_T = -128.0;
```

Comment:
These lines were only moved. But now they are between vars used for PM sensors ...

Reply:
good point, fixed that. the real change is double to float though.. (to reduce float/double conversions as those values are captured as floats by the sensor libraries)